src/Mutator/Util/MutatorProfile.php

```diff
@@ -63,13 +65,11 @@
     const BOOLEAN = [
         Mutator\Boolean\ArrayItem::class,
         Mutator\Boolean\FalseValue::class,
-        Mutator\Boolean\IdenticalEqual::class,
```

Comment:
These are particulary useful mutators. E.g. it is very easy to forget to test an example below on the inclusion of a substring at the beginning of the string:
```
false !== strpos($string, $substring)
```
When this example is mutated, then it could skip under the radar if a test coverage is insufficient:
```
false != strpos($string, $substring)
```
Otherwise put, if this mutation lives on, your tests are not good enough.

Removal of these mutators from the default profile is unwarranted.

Reply:
This comes down to how you write your code. If you default to `==`, and only use `===` when strictly needed, than this mutator is rather useful.

However, if you default to `===` and only use `==` when strictly needed, 90+% of these mutations can't be caught.

For a lot of projects, the default is `===`. For this reason PHP-CS-Fixer has the `strict_comparison` fixer, that changes `==` to `===`, and coding standards like the doctrine one disallow usage of `==`.

Reply:
This mutator was created specifically to cover practical and useful cases like above and similar. It has nothing to do with coding styles.

If you think we should be using `===` at all time, best course of action is to propose adding `strict_comparison` fixer. Removal of an essential mutator solves nothing.